### DISCUSSION TITLE
Add spacing to UniformItemsLayout

### DISCUSSION THREAD

**Discussion by MauiUIui** [CommunityToolkit/Maui#1152] [General]
I really like `UniformGrid` from WPF and I'm very thankful you guys made the effort to make the functionality also available for MAUI. However doesn't it make sense to make it more MAUI-compatible by including a "`Spacing`" property. For reference WPF doesn't have spacing in any of its panels, including its most improtant one (`Grid`), but MAUI does have `RowSpacing` and `ColumnSpacing` built-in in the `Grid` and `Spacing` in other layouts.

_Originally posted by @MauiUIui in https://github.com/CommunityToolkit/Maui/issues/122#issuecomment-1509377648_

**Comment by BaY1251** [comment]
+1

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889

**Comment by bijington** [comment]
@MauiUIui as I mentioned from your previous discussion I have reopened this discussion.

Now onto your suggestion. Do you have an accurate proposal of what it is that you would like to see added here? I suspect there are a couple of options:
- Add a single `Spacing` property.
- Add a `RowSpacing` and `ColumnSpacing` property.

Also for this functionality to be added would you be a willing contributor for any/all of the requirements that we expect:
- Code changes
- Unit test additions
- Addition of the functionality in to the sample application
- Update the documentation

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers. 

If you feel this is being closed in error please let us know and we can re-open.
